src/AppBundle/Charts/Charts.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace AppBundle\Charts;
+
+use Ob\HighchartsBundle\Highcharts\Highchart;
+use AppBundle\RequestSql\RequestSql; 
+
+class Charts
+{
+    private $xAxis;
+    private $yAxis;
+    private $legend;
+    private $requestSql;
+    private $typeGraph;
+    protected $rqSql;
+
+    /***
+     * Charts constructor.
+     * @param RequestSql $rqSql
+     */
+    public function __construct(RequestSql $rqSql)
+    {
+        $this->xAxis      = "X axis name";
+        $this->yAxis      = "Y axis name";
+        $this->legend     = "Legend name";
+        $this->requestSql = array('');
+        $this->typeGraph  = "column";
+        $this->rqSql      = $rqSql;
+    }
+
+    /***
+     * @param $value
+     */
+    public function setXAxis($value){
+        $this->xAxis = $value;
+    }
+
+    /***
+     * @param $value
+     */
+    public function setYAxis($value){
+        $this->yAxis = $value;
+    }
+
+    /***
+     * @param $value
+     */
+    public function setLegend($value){
+        $this->legend = $value;
+    }
+
+    /***
+     * @param $value
+     */
+    public function setRequestSql($value){
+        $this->requestSql = $value;
+    }
+
+    /***
+     * @param $value
+     */
+    public function setTypeGraph($value){
+        $this->typeGraph = $value;
+    }
+
+    /***
+     * @return string
+     */
+    public function getXAxis(){
+        return $this->xAxis;
+    }
+
+    /***
+     * @return string
+     */
+    public function getYAxis(){
+        return $this->yAxis;
+    }
+
+    /***
+     * @return string
+     */
+    public function getLegend(){
+        return $this->legend;
+    }
+
+    /***
+     * @return array
+     */
+    public function getRequestSql(){
+        return $this->requestSql;
+    }
+
+    /***
+     * @return string
+     */
+    public function getTypeGraph(){
+        return $this->typeGraph;
+    }
+
+    /***
+     * @param $key
+     * @return Highchart
+     */
+    public function generateChart($key){
```

Comment:
What do you think to pass above attributes as parameters of this function instead to have to construct this class with default parameters?